baker/algolia/configureAlgolia.ts

```diff
@@ -109,6 +109,23 @@ export const configureAlgolia = async () => {
         disableExactOnAttributes: ["tags"],
     })
 
+    const explorersIndex = client.initIndex("explorers")
+
+    await explorersIndex.setSettings({
+        ...baseSettings,
+        searchableAttributes: [
+            "unordered(slug)",
+            "unordered(title)",
+            "unordered(subtitle)",
+            "unordered(views_7d)",
+            "unordered(text)",
+        ],
+        customRanking: ["desc(score)", "desc(importance)"],
+        attributesToSnippet: ["excerpt:20", "content:20"],
```

Comment:
```suggestion
        attributesToSnippet: ["subtitle:24"],
```

Reply:
@marcelgerber [Ike seemed to be thinking that we shouldn't show subtitles for now](https://github.com/owid/owid-grapher/blob/e00f3f619e10a3b6aefcbaa1ec5006460de2d914/site/search/SearchPanel.tsx#L125). Let's revisit later?

Reply:
Makes sense! Doesn't really matter whether we already include it (and don't use it) or drop it entirely 👍🏻